references/auth-schemes/apiproxy/policies/AssignMessage.SetPrivateKey.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2021 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<AssignMessage continueOnError="false" enabled="true" name="AssignMessage.SetPrivateKey">
+    <AssignVariable>
+        <Name>private.key</Name>
+        <!-- this private key is not and must not be used elsewhere -->
```

Comment:
..is not [private/secure] and must..?

Reply:
I'm trying to say this private key `(is not) AND (must not)` be used elsewhere, but I agree it reads weird...